src/content/docs/dropins/payment-services/installation.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Payment Services installation
+description: Learn how to install the Payment Services drop-in component on your site.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+The Payment Services drop-in component allows users to efficiently oversee payment and order data from all of your storefronts in one place — your Commerce dashboard. By leveraging Payment Services, merchants can focus on growing their business while providing their customers with a smooth and trustworthy checkout experience.
+
+## Installation
+
+The Payment Services component relies on containers from several other drop-in components, you must install and configure those components before you can use the Payment Services component to place an order.
+
+The [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce) includes all of the necessary drop-in components and configurations to help you get started quickly, so Adobe recommends relying on the boilerplate instead of installing, configuring, and integrating the drop-in components individually.
+
+## Admin configuration
+
+Before you can use the Payment Services component on your storefront, you must [enable](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/user-guides/integration-services/saas) and [configure the Services Connector](https://experienceleague.adobe.com/en/docs/commerce-learn/tutorials/admin/adobe-commerce-services/configure-adobe-commerce-services-connector) in the Adobe Commerce Admin.
+
```

Comment:
To use Payment Services, merchants also have to go through onboarding - https://experienceleague.adobe.com/en/docs/commerce-merchant-services/payment-services/get-started/onboard
Just enabling and configuring Services Connector is not enough.

Reply:
Thanks @AVoskoboinikov !, I'll mention too about the minimum version of the extension